### DISCUSSION TITLE
[Bug] Errors...

### DISCUSSION THREAD

**Discussion by ghost** [U61vashka/MC-Auth-with-VK#200] [Q&A]
Когда запускаю BungeeCord выдает это:
18:20:32 [SEVERE] com.bivashy.auth.lib.org.spongepowered.configurate.loader.ParsingException: []: Unknown error occurred while loading

18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.spongepowered.configurate.loader.AbstractConfigurationLoader.load(AbstractConfigurationLoader.java:165)

18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.spongepowered.configurate.loader.AbstractConfigurationLoader.load(AbstractConfigurationLoader.java:63)
                                                                  18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.spongepowered.configurate.loader.ConfigurationLoader.load(ConfigurationLoader.java:56)

18:20:32 [SEVERE]     at me.mastercapexd.auth.config.SpongeConfiguratePluginConfig.getConfigurationNode(SpongeConfiguratePluginConfig.java:71)

18:20:32 [SEVERE]     at me.mastercapexd.auth.config.SpongeConfiguratePluginConfig.loadConfiguration(SpongeConfiguratePluginConfig.java:32)

18:20:32 [SEVERE]     at me.mastercapexd.auth.config.BasePluginConfig.createConfiguration(BasePluginConfig.java:15)

18:20:32 [SEVERE]     at me.mastercapexd.auth.config.PluginConfigTemplate.<init>(PluginConfigTemplate.java:109)

18:20:32 [SEVERE]     at me.mastercapexd.auth.config.SpongeConfiguratePluginConfig.<init>(SpongeConfiguratePluginConfig.java:24)    
18:20:32 [SEVERE]     at me.mastercapexd.auth.config.BasePluginConfig.<init>(BasePluginConfig.java:10)

18:20:32 [SEVERE]     at me.mastercapexd.auth.BaseAuthPlugin.initializeBasic(BaseAuthPlugin.java:145)                               
18:20:32 [SEVERE]     at me.mastercapexd.auth.BaseAuthPlugin.<init>(BaseAuthPlugin.java:131)                                        
18:20:32 [SEVERE]     at me.mastercapexd.auth.bungee.BungeeAuthPluginBootstrap.onEnable(BungeeAuthPluginBootstrap.java:37)

18:20:32 [SEVERE]     at net.md_5.bungee.api.plugin.PluginManager.enablePlugins(PluginManager.java:266)

18:20:32 [SEVERE]     at net.md_5.bungee.BungeeCord.start(BungeeCord.java:298)

18:20:32 [SEVERE]     at net.md_5.bungee.BungeeCordLauncher.main(BungeeCordLauncher.java:67)

18:20:32 [SEVERE]     at net.md_5.bungee.Bootstrap.main(Bootstrap.java:15)

18:20:32 [SEVERE] Caused by: com.bivashy.auth.lib.org.yaml.snakeyaml.error.YAMLException: java.nio.charset.MalformedInputException: Input length = 1
                                                                  18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.yaml.snakeyaml.reader.StreamReader.update(StreamReader.java:211)

18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.yaml.snakeyaml.reader.StreamReader.ensureEnoughData(StreamReader.java:169)

18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.yaml.snakeyaml.reader.StreamReader.peek(StreamReader.java:129)

18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.spongepowered.configurate.yaml.ConfigurateScanner.scanPlain(ConfigurateScanner.java:2016)                                                           
18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.spongepowered.configurate.yaml.ConfigurateScanner.fetchPlain(ConfigurateScanner.java:1073)
                                                                  18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.spongepowered.configurate.yaml.ConfigurateScanner.fetchMoreTokens(ConfigurateScanner.java:428)

18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.spongepowered.configurate.yaml.ConfigurateScanner.checkToken(ConfigurateScanner.java:256)                                                           
18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.yaml.snakeyaml.parser.ParserImpl$ParseBlockMappingKey.produce(ParserImpl.java:662)

18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.yaml.snakeyaml.parser.ParserImpl.peekEvent(ParserImpl.java:185)

18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.yaml.snakeyaml.comments.CommentEventsCollector$1.peek(CommentEventsCollector.java:57)                                                               
18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.yaml.snakeyaml.comments.CommentEventsCollector$1.peek(CommentEventsCollector.java:43)
18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.yaml.snakeyaml.comments.CommentEventsCollector.collectEvents(CommentEventsCollector.java:136)

18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.yaml.snakeyaml.comments.CommentEventsCollector.collectEvents(CommentEventsCollector.java:116)
                                                                  18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.yaml.snakeyaml.composer.Composer.composeScalarNode(Composer.java:239)
                                                                  18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.yaml.snakeyaml.composer.Composer.composeNode(Composer.java:208)

18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.yaml.snakeyaml.composer.Composer.composeValueNode(Composer.java:357)
                                                                  18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.yaml.snakeyaml.composer.Composer.composeMappingChildren(Composer.java:336)

18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.yaml.snakeyaml.composer.Composer.composeMappingNode(Composer.java:311)

18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.yaml.snakeyaml.composer.Composer.composeNode(Composer.java:212)                   
18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.yaml.snakeyaml.composer.Composer.composeValueNode(Composer.java:357)              
18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.yaml.snakeyaml.composer.Composer.composeMappingChildren(Composer.java:336)

18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.yaml.snakeyaml.composer.Composer.composeMappingNode(Composer.java:311)

18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.yaml.snakeyaml.composer.Composer.composeNode(Composer.java:212)                   
18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.yaml.snakeyaml.composer.Composer.composeValueNode(Composer.java:357)
                                                                  18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.yaml.snakeyaml.composer.Composer.composeMappingChildren(Composer.java:336)        
18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.yaml.snakeyaml.composer.Composer.composeMappingNode(Composer.java:311)

18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.yaml.snakeyaml.composer.Composer.composeNode(Composer.java:212)

18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.yaml.snakeyaml.composer.Composer.getNode(Composer.java:134)                       
18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.yaml.snakeyaml.composer.Composer.getSingleNode(Composer.java:160)

18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.yaml.snakeyaml.constructor.BaseConstructor.getSingleData(BaseConstructor.java:184)
                                                                  18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.spongepowered.configurate.yaml.ConfigurateYaml.loadConfigurate(ConfigurateYaml.java:39)

18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.spongepowered.configurate.yaml.YamlConfigurationLoader.loadInternal(YamlConfigurationLoader.java:158)

18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.spongepowered.configurate.yaml.YamlConfigurationLoader.loadInternal(YamlConfigurationLoader.java:42)

18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.spongepowered.configurate.loader.AbstractConfigurationLoader.load(AbstractConfigurationLoader.java:155)

18:20:32 [SEVERE]     ... 15 more

18:20:32 [SEVERE] Caused by: java.nio.charset.MalformedInputException: Input length = 1
                                                                  18:20:32 [SEVERE]     at java.base/java.nio.charset.CoderResult.throwException(CoderResult.java:274)                                
18:20:32 [SEVERE]     at java.base/sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:326)

18:20:32 [SEVERE]     at java.base/sun.nio.cs.StreamDecoder.read(StreamDecoder.java:188)

18:20:32 [SEVERE]     at java.base/java.io.InputStreamReader.read(InputStreamReader.java:177)

18:20:32 [SEVERE]     at java.base/java.io.BufferedReader.fill(BufferedReader.java:162)
18:20:32 [SEVERE]     at java.base/java.io.BufferedReader.read1(BufferedReader.java:213)

18:20:32 [SEVERE]     at java.base/java.io.BufferedReader.read(BufferedReader.java:287)

18:20:32 [SEVERE]     at com.bivashy.auth.lib.org.yaml.snakeyaml.reader.StreamReader.update(StreamReader.java:176)

18:20:32 [SEVERE]     ... 48 more                                 
18:20:32 [WARNING] Exception encountered when loading plugin: mcAuth        java.lang.NullPointerException
    at java.base/java.util.Objects.requireNonNull(Objects.java:208)
    at com.bivashy.configuration.configurate.holder.ConfigurationNodeHolder.<init>(ConfigurationNodeHolder.java:20)
    at me.mastercapexd.auth.config.BasePluginConfig.createConfiguration(BasePluginConfig.java:15)
    at me.mastercapexd.auth.config.PluginConfigTemplate.<init>(PluginConfigTemplate.java:109)
    at me.mastercapexd.auth.config.SpongeConfiguratePluginConfig.<init>(SpongeConfiguratePluginConfig.java:24)
    at me.mastercapexd.auth.config.BasePluginConfig.<init>(BasePluginConfig.java:10)
    at me.mastercapexd.auth.BaseAuthPlugin.initializeBasic(BaseAuthPlugin.java:145)
    at me.mastercapexd.auth.BaseAuthPlugin.<init>(BaseAuthPlugin.java:131)
    at me.mastercapexd.auth.bungee.BungeeAuthPluginBootstrap.onEnable(BungeeAuthPluginBootstrap.java:37)
    at net.md_5.bungee.api.plugin.PluginManager.enablePlugins(PluginManager.java:266)
    at net.md_5.bungee.BungeeCord.start(BungeeCord.java:298)
    at net.md_5.bungee.BungeeCordLauncher.main(BungeeCordLauncher.java:67)
    at net.md_5.bungee.Bootstrap.main(Bootstrap.java:15)

Как исправить?

**Comment by bivashy** [comment]
Некорректная конфигурация.

**Reply by ghost** [reply]
Странно. Но я все настроил

**Reply by bivashy** [reply]
> Странно. Но я все настроил

Дело не в настройке, а в том что конфигурация может иметь некорректный формат (Вместо UTF-8), либо что то подобное. 
То есть проблема может быть в формате в котором вы сохранили файл.
